core/templates/dev/head/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.html

```diff
@@ -12,18 +12,36 @@
       <i class="material-icons">play_arrow</i>
     </button>
   </div>
-  <div class="oppia-content-wrapper oppia-audio-bar-info">
-    <div ng-if="!cannotRecord && !checkingMicrophonePermission">
-      No audio recorded.<span ng-if="canVoiceover"> Press <i class="material-icons">mic</i>  to start recording.</span>
-    </div>
-    <div ng-if="checkingMicrophonePermission">
-      Loading microphone.<loading-dots></loading-dots>
-    </div>
-    <div ng-if="recordingPermissionDenied && cannotRecord">
-      You need to grant permission for this application to use your microphone.
+  <div class="d-block d-xl-none oppia-content-wrapper oppia-audio-bar-info">
+    <div class="padding-small-screen">
+      <div ng-if="!cannotRecord && !checkingMicrophonePermission">
+        <span ng-if="canVoiceover"> Press <i class="material-icons">mic</i>  to record.</span>
+      </div>
+      <div ng-if="checkingMicrophonePermission">
+        Loading microphone.<loading-dots></loading-dots>
+      </div>
+      <div ng-if="recordingPermissionDenied && cannotRecord">
+        Grant permission to record.
```

Comment:
Maybe change to 'Permission needed for recording.'? Does that sound good? If it just overflows, try reducing font size a bit. The other messages seem fine.

Reply:
I have changed the wordings and reduced the  font size.
![Screenshot from 2020-01-14 08-01-30](https://user-images.githubusercontent.com/42460632/72309546-3ed6f500-36a5-11ea-9f8e-7ea2edeb3947.png)